Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml

```diff
@@ -266,14 +266,14 @@
                             </div>
                             <div class="govuk-date-input__item">
                                 <div class="govuk-form-group">
-                                    <label for="amalgamationdate-month" class="govuk-label govuk-date-input__label">Month</label>
-                                    <input class="govuk-input date-text-month" type="text" v-model="mergeDateMonth" id="amalgamationdate-month">
+                                    <label for="amalgamation-date-month" class="govuk-label govuk-date-input__label">Month</label>
```

Comment:
@JaswiniP should the for value be changed here? FYI @RogerHowellDfE

Reply:
My best understanding - this value needs to match the model class

Reply:
The `for` attribute of a `<label>` needs to match the `id` value of the `<input>` element the label is describing - updating one needs the other to be updated also, so updating both here is fine :)